modules/user/client-react/components/ActivateUser.web.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate } from '@module/i18n-client-react';
+import { LayoutCenter } from '@module/look-client-react';
+
+class ActivateUser extends React.PureComponent {
```

Comment:
Can you use Function component instead of Class component?

Reply:
Done